pom.xml

```diff
@@ -32,7 +32,7 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.5</version>
+            <version>2.6.2</version>
```

Comment:
We need to update Gson since 2.6 is the first series that allows a `String` to be a top-level Json text.